tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -187,7 +187,7 @@ $VENV_PYTHON $ROOT/src/python/grpcio_reflection/setup.py build_package_protos
 pip_install_dir $ROOT/src/python/grpcio_reflection
 
 # Build/install tests
-$VENV_PYTHON -m pip install coverage oauth2client
+$VENV_PYTHON -m pip install coverage oauth2client google-auth requests
```

Comment:
We don't have these pinned? Huh. Okay. Is now the time to think about pinning them?